pkgs/development/python-modules/threadloop/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, buildPythonPackage, fetchPypi
+, tornado
+, flake8, ipdb, pytest, pytest-cov, twine, zest_releaser, coveralls
+}:
+
+buildPythonPackage rec {
+  pname = "threadloop";
+  version = "1.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "8b180aac31013de13c2ad5c834819771992d350267bddb854613ae77ef571944";
+  };
+
+  propagatedBuildInputs = [
+    tornado
+  ];
+
+  checkInputs = [
+    flake8
+    ipdb
+    pytest
+    pytest-cov
+    twine
+    zest_releaser
+    coveralls
+  ];
+  checkPhase = ''
+    pytest
```

Comment:
What about `pytestCheckHook`?

Reply:
Tests are not working because there are missing dependencies.